### ISSUE TITLE
ngInject does not work on direct export

### ISSUE THREAD

**Issue by clemgrim** [schmod/babel-plugin-angularjs-annotate#24]
Hello,

ngInject does not work when exporting a class, it worked with ng-annotate.

```js
/* @ngInject */
class A {
    constructor(test){
      this.test = test;
    }
}
```

This works.

```js
/* @ngInject */
export class A {
    constructor(test){
      this.test = test;
    }
}
```
This doesn't.

```js
/* @ngInject */
class A {
    constructor(test){
      this.test = test;
    }
}

export {A};
```

This does.

**Comment by schmod** [comment]
Interesting.

While I look into this, there are a few workarounds you can use:

```
export class A {
    /* @ngInject */
    constructor(test){
      this.test = test;
    }
}
```

and

```
export class A {
    constructor(test){
      'ngInject';
      this.test = test;
    }
}
```

will both work.  

In general, the second syntax (prologue directives) is going to be more reliable, as other tools/transformations should leave these untouched (whereas comments are frequently moved around or mangled by other transformations).

**Comment by schmod** [comment]
fixed in 0.8.0
